src/router.js

```diff
@@ -80,15 +80,14 @@ export class Router {
   * @param viewPort The viewPort.
   * @param name The name of the viewPort. 'default' if unspecified.
   */
-  registerViewPort(viewPort:Object, name?:string) {
-    name = name || 'default';
+  registerViewPort(viewPort : Object, name? : string = 'default') : void {
```

Comment:
I'm a little nervous about this change. I think templating-router may end up passing empty string to this, and this change would break that. Have you checked out existing callers of this method?